.travis/script.sh

```diff
@@ -40,7 +40,7 @@ case "${BUILD_TARGET}" in
 
     linux-auto-gcc-check-happy)
         # run happy test
-        sudo bash -c "source ${HOME}/ve/happy/bin/activate; make -f Makefile-Standalone DEBUG=1 TIMESTAMP=1 COVERAGE=1 BuildJobs=24 BLUEZ=1 check"
```

Comment:
Why did we remove BLUEZ here? Do the tests no longer work or run?

Reply:
The bluez test is failing after change to non virtual python env in setup part, I need to debug and fix it.
I have created an issue to track, I will address it in a different PR.
https://github.com/openweave/openweave-core/issues/166

Reply:
OK, thanks.